### PR TITLE
Resdiff Integration: Linting and clean up (Part I)

### PR DESCRIPTION
**Pull request by mnabian**
# Modulus Pull Request

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [x] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

**Review by MortezaMardani — APPROVED** [review]

**Review by daviddpruitt — APPROVED** [review]

File: docs/test_runner.py [hunk 1]
```diff
@@ -4,9 +4,9 @@
 import pytest
 
 # Collecting all the Python files in the scripts directory
-script_files = [f for f in os.listdir("test_scripts/") if f.endswith('.py')]
+script_files = [f for f in os.listdir("test_scripts/") if f.endswith(".py")]
```
Comment (NickGeneva) [inline docs/test_runner.py:7]: Just checking: Is this from the formatter? What version of black do you have installed? I'm assuming this was done via pre-commit?

File: modulus/experimental/resdiff/dataset_tool.py [hunk 2]
```diff
@@ -186,17 +186,22 @@ def open_cifar10(tarball: str, *, max_images: Optional[int]):
...
 
     images = np.concatenate(images)
     labels = np.concatenate(labels)
     images = images.transpose([0, 2, 3, 1])  # NCHW -> NHWC
-    assert images.shape == (50000, 32, 32, 3) and images.dtype == np.uint8
-    assert labels.shape == (50000,) and labels.dtype in [np.int32, np.int64]
-    assert np.min(images) == 0 and np.max(images) == 255
-    assert np.min(labels) == 0 and np.max(labels) == 9
+    if not (images.shape == (50000, 32, 32, 3) and images.dtype == np.uint8):
+        raise RuntimeError("Unexpected image shape or type")
```
Comment (NickGeneva) [inline modulus/experimental/resdiff/dataset_tool.py:198]: Nit: Probably should be ValueError

File: docs/test_runner.py (hunk 1 above)
Comment (mnabian) [reply docs/test_runner.py:7]: Yes, this is from the formatter. I'm using `22.10.0`

File: modulus/experimental/resdiff/dataset_tool.py (hunk 2 above)
Comment (mnabian) [reply modulus/experimental/resdiff/dataset_tool.py]: Thanks, fixed it.

**Pull request merged**